src/styles/card.scss

```diff
@@ -0,0 +1,39 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
+    margin: 0 16px 4px;
+    font-family: $font-roboto;
+    font-weight: $font-medium;
+    color: $color-font;
+  }
```

Comment:
The fixed width and height for `.card__title` might cause issues with text overflow or clipping if the content is longer than the space provided. Consider using `min-height` instead of `height` and ensure that the container can handle variable text lengths.